staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -198,6 +198,9 @@ type Config struct {
 	// kube-proxy, services, etc.) can reach the GenericAPIServer.
 	// If nil or 0.0.0.0, the host's default interface will be used.
 	PublicAddress net.IP
+	// The port on which to advertise the apiserver to members of the cluster. If
+	// blank, the bind port of the SecureServing listener will be used.
```

Comment:
This doc isn't really true anymore now that we moved defaulting into server_run_options.go